src/cairo_vm.rs

```diff
@@ -36,7 +35,7 @@ pub fn cairo_run(
     cairo_run_config: &CairoRunConfig,
     hint_executor: &mut dyn HintProcessor,
     variables: HashMap<String, Box<dyn Any>>,
-) -> Result<(CairoRunner, VirtualMachine), CairoRunError> {
```

Comment:
This whole function can be removed and replaced by `cairo_run_program_with_initial_scope`.